translate/storage/cpo.py

```diff
@@ -735,7 +741,7 @@ def isempty(self):
                 return False
         return True
 
-    def parse(self, input):
+    def parse(self, input, duplicatestyle=None):
```

Comment:
Please provide a default value. The same goes for the other two implementations of this method.